docker-compose.yml

```diff
@@ -49,6 +49,8 @@ services:
       context: client
       dockerfile: Dockerfile
     container_name: phi-deidentifier-app
+    environment:
+      - SERVER_PORT=80
```

Comment:
We should be able to specify here the SERVER_SCHEME and SERVER_HOST, no?

Reply:
Yes, we can, but the defaults in [`.env`](https://github.com/nlpsandbox/phi-deidentifier/blob/6a099bb52f2dae17dfae5970b6daac3215ca24e6/client/.env) are already what we want them to be. Should I include the same values in `docker-compose.yml` anyway?

Reply:
Yes, let's be more explicit. You can add variables using this synthax: `VARIABLE=${VARIABLE}` where the value `${VARIABLE}` will be read automatically from `.env` by docker-compose.

Reply:
Something that might not be clear: the file that I had named `client/.env` has nothing to do with the `.env` used to provide values for environmental variables in `docker-compose`. Therefore I renamed the file to `env_vars`.

`client/env_vars` gets read by a script, `client/env.sh`, which then outputs env var values (defaulting to the values in `client/env_vars`) to a static file `env-config.js`, which gets served (statically). Like I said at the top of this PR, it is very weird and convoluted, but it works.

If you want to override the default values in `client/env_vars`, you can do so in the normal way, by putting them in a `.env` file in the project root. I have tested that this works. Does this all make sense?

Reply:
Sort of but not completely. As long as the docker-compose.yml allows us to specify the port, host, etc. we are good.